package.json

```diff
@@ -1,44 +1,75 @@
 {
-  "name": "webpack-react-practice",
+  "name": "FuBoFleaMarket",
   "version": "1.0.0",
-  "description": "First react project",
+  "description": "A flea market web site for fu-bo",
   "main": "app/app.js",
   "scripts": {
-    "test": "test",
     "build": "webpack",
+    "eslint": "eslint --fix --ext .js app test",
     "start": "webpack-dev-server --devtool eval --progress --colors --hot --content-base build",
-    "eslint": "eslint --ext .js src"
+    "test": "cross-env NODE_ENV=test mocha --recursive --compilers js:babel-register --require ./test/setup.js --reporter mochawesome",
+    "test:watch": "npm test -- --watch"
   },
-  "author": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Fu-Bo-Tech/fubo-flea-market.git"
+  },
+  "author": "ben196888, ElaineHuang",
   "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Fu-Bo-Tech/fubo-flea-market/issues"
+  },
+  "homepage": "https://github.com/Fu-Bo-Tech/fubo-flea-market#readme",
   "devDependencies": {
-    "autoprefixer-loader": "^3.2.0",
```

Comment:
This module is deprecated. Autoprefixer official page recommends using postcss-loader instead.
postcss-loader is pure css loader, so we remove autoprefixer-loader and not install postcss-loader